crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs

```diff
@@ -271,26 +381,56 @@ fn split_recipients_withhelds_for_user_based_on_identity(
             RecipientDevices {
                 allowed_devices: recipients,
                 denied_devices_with_code: withheld_recipients,
+                unsigned_of_verified_user: Vec::default(),
             }
         }
     }
 }
 
+fn is_unsigned_device_of_verified_user(
+    own_identity: &Option<OwnUserIdentityData>,
+    device_owner_identity: &Option<UserIdentityData>,
```

Comment:
:crab: nit: usually (and I know there's a clippy lint for this, not enabled in our code base), it's preferable to use `Option<&T>` instead of `&Option<T>` in parameters lists, because that's how we're going to use it later. (Proof in case, you're calling `as_ref()` in the bodies, and following my suggestion it would be called at the call-sites instead.)

Reply:
Yup. Done in b341e19e0.